mypaint-brush.c

```diff
@@ -22,6 +22,18 @@
 #include <math.h>
 #include <assert.h>
 
+#define GETTEXT_PACKAGE "libmypaint"
+
+#ifdef HAVE_GETTEXT
```

Comment:
I think so, but maybe that should be a separate change to modify everything and we can leave it to @achadwick to decide if that change is wanted.